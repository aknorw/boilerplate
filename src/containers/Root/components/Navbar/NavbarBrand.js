import styled from 'styled-components'
import { Link } from 'react-router-dom'

export default styled(Link)`
  color: white;
  text-decoration: none;
  cursor: pointer;
  -webkit-font-smoothing: antialiased;
  font-size: 1.25em;
  font-weight: bold;
  text-transform: uppercase;
`
