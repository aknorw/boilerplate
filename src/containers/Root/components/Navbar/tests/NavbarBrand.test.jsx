import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import NavbarBrand from '../NavbarBrand'

console.error = jest.fn()

describe('<NavbarBrand />', () => {
  it('should be defined', () => {
    expect(NavbarBrand).toBeDefined()
  })
  it('should log 1 error when `to` is missing', () => {
    shallow(<NavbarBrand />)
    expect(console.error).toHaveBeenCalledTimes(1)
  })
  it('should render correctly', () => {
    const tree = shallow(
      <NavbarBrand />,
    )
    expect(tree).toMatchSnapshot()
  })
})
