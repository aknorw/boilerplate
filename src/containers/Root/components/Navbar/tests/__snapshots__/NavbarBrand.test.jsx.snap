// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarBrand /> should render correctly 1`] = `
.c0 {
  color: white;
  -webkit-text-decoration: none;
  text-decoration: none;
  cursor: pointer;
  -webkit-font-smoothing: antialiased;
  font-size: 1.25em;
  font-weight: bold;
  text-transform: uppercase;
}

<Link
  className="c0"
  replace={false}
/>
`;
